ui/src/AzureUserInput.ts

```diff
@@ -18,21 +18,25 @@ export class AzureUserInput implements IAzureUserInput {
         this._persistence = persistence;
     }
 
-    public async showQuickPick<T extends QuickPickItem>(items: T[] | Thenable<T[]>, options: QuickPickOptions): Promise<T> {
+    public async showQuickPick<T extends QuickPickItem>(items: T[] | Thenable<T[]>, options: QuickPickOptions & { canPickMany: boolean }): Promise<T | T[]> {
```

Comment:
nit: Remove `{ canPickMany: boolean }`. It's already a part of QuickPickOptions and unlike index.d.ts, you're not setting it to a specific value in this case